src/server-stub/main.rs

```diff
@@ -35,12 +35,28 @@ impl TokenService for Service {
         Ok(Response::new(reply))
     }
 
+    async fn enroll_cluster_agent(
+        &self,
+        request: Request<pb::EnrollClusterAgentRequest>,
+    ) -> Result<Response<pb::EnrollClusterAgentResponse>, Status> {
+
+        println!("enroll_cluster_agent: {:?}", request.into_inner());
+
+        let expiration = SystemTime::now() + Duration::from_secs(180);
```

Comment:
Is there anything special about 3 minutes?

Reply:
No but when testing, I don't want to wait too long.